src/main/java/com/alex/eshop/service/ItemService.java

```diff
@@ -34,4 +45,24 @@ public List<ItemDTO> getLastFiveItems() {
     public List<ItemDTO> getItemsInCategory(Long categoryId) {
         return itemMapper.toDto(itemRepository.findByCategoryId(categoryId));
     }
+    public Item createItem(ItemCreateDTO itemCreateDTO){
+        return itemRepository.save(itemCreateMapper.toEntity(itemCreateDTO));
+    }
+
+    public void updateItem(ItemUpdateDTO itemUpdateDTO){
+        ItemUpdateDTO itemUpdate = itemUpdateMapper
+                .toDto(itemRepository.findById(itemUpdateDTO.getId())
```

Comment:
For PUT operation there is not need to search existing item in db. Just update whole entity as is.
You can read about difference between PUT and PATCH methods. For PATCH you will need to search in DB, for PUT - not.